OrbitCore/ElfFile.cpp

```diff
@@ -128,6 +133,7 @@ bool ElfFileImpl<ElfT>::GetFunctions(Pdb* pdb,
     }
 
     std::string name = symbol_ref.getName() ? symbol_ref.getName().get() : "";
+    std::string pretty_name = name;
 
 #if __linux__
     name = LinuxUtils::Demangle(name.c_str());
```

Comment:
Shouldn't be the pretty name the demangled one?

Reply:
it should - fixed.